src/app/(dashboard)/residents/[id]/page.tsx

```
@@ -829,20 +677,47 @@
+                fontWeight: '500',
+                color: '#4b5563',
+              }}
             >
-              <svg className="mr-2 size-4" fill="none" stroke="currentColor" viewBox="0 0 24 24" style={{ marginRight: '8px', width: '16px', height: '16px' }}>
+              <svg
+                className="mr-2 size-4"
+                fill="none"
+                stroke="currentColor"
+                viewBox="0 0 24 24"
+                style={{ marginRight: '8px', width: '16px', height: '16px' }}
+              >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)